### DISCUSSION TITLE
0 stock :( Pyboard D-series with STM32F722 and WiFi/BT

### DISCUSSION THREAD

**Discussion by Eun0us** [micropython/micropython#16486] [General]
Hello, I've been actively looking at this card's stock for 1 year, and there's no stock left. I need it for a study project. 
Do you have the idea to restock or i have to search on other market place

**Comment by jonnor** [comment]
I do not have any inside information as to what the situation is. But I do not think you should expect any stock to appear any time soon. However, for many years already, MicroPython runs on many boards - you are not at all limited to a Pyboard. So I would recommend you to get a board from another hardware vendor, and flash MicroPython on that. Use the download page to see what is supported.
